k8s/edge/k8s-template

```diff
@@ -94,36 +89,46 @@ if __name__ == '__main__':
                         help='Command to reload configuration', required=False)
     parser.add_argument('--once', '-n', action='store_true',
                         help='Render template Once', required=False)
-    parser.add_argument('--pause', '-p', help='Pause length, in seconds', type=int,
-                        default=5, required=False)
+    parser.add_argument('--required-label', '-p', help='Pause length, in seconds', type=str,
+                        default='', required=False)
```

Comment:
`required=False` useless

Reply:
i'll fix it next week.

Reply:
and default value for non-required parameters is None

Reply:
Done